Tests/FluentAssertions.Specs/Primitives/HttpResponseMessageAssertionSpecs.cs

```diff
@@ -0,0 +1,261 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Primitives
+{
+    public class HttpResponseMessageAssertionSpecs
+    {
+        [Theory]
+        [MemberData(nameof(GetSuccessStatusCodes))]
```

Comment:
🤔 I don't think it's very useful to run this test for every possible value. Just taking some sensible values, including edge cases would be much easier to understand. This also avoids the need to have a helper method, which I personally feel obfuscates the test purpose. Also applies to the other tests.